### DISCUSSION TITLE
Templater Calendar Question

### DISCUSSION THREAD

**Discussion by ddetton** [BryanJenksCommunity/FAQ#58] [Q&A] [answered]
Hey Bryan, I converted my Obsidian daily note template to templater and ran into a little problem that I couldn't figure out so I reverted back to Calendar functions.  Sometimes I forget to write my daily note.  When this happens I will open the calendar and click on the day that I missed and it creates the daily note for that day.  When I tried to do this with templater, I couldn't figure out how to get the date that I clicked in the calendar.  It worked fine I was creating the note for today but not for any other day where it needed to create the file.  Do you see a way to do that in templater?  Here is what I have now that does what I want using the Calendar functions:

[[{{date-1d:YYYY-MM-DD}}]]	  <-	Today	->	  [[{{date+1d:YYYY-MM-DD}}]]

# Daily Note for {{date:dddd, MMMM D, YYYY}}

**Comment by tallguyjenks** [comment, marked as answer]
Here you go :)

```md
[[<%tp.date.now("YYYY-MM-DD", -1)%>]] <- Today -> [[<%tp.date.now("YYYY-MM-DD", 1)%>]]

<%tp.date.now("dddd, MMMM D, YYYY")%>
```

**Reply by ddetton** [reply]
Yeah, that's what I originally thought too.  Here is my use case.  I have Liam's calendar open and I forgot to enter the daily note for Sunday, April 11, so I click on April 11 in the calendar to create that daily note.  When I have my original date functions, in my Daily Note Template the daily note for April 11 is created with Sunday, April 11 as my main heading in the note.  When I use the function with tp.date.now("dddd, MMMM D, YYYY"), it creates the April 11 daily note, but the date that is printed inside the  note is today's date 4-13-2021.  Try it and let me know if you get different results.

I tried it with Periodic Notes plug in enabled and Daily Note core disabled and vise-versa and get same results.

**Reply by tallguyjenks** [reply]
yeah the tempaltes all input the current date `tp.date.now` so that going to be the behavior. Perosnally i just make the daily notes every day so I never have an issue, if I needed to I don't mind manually correcting one.

i don't know if this will work but using the calendar date fields with the `gggg-MM-DD` date string might help it might not. i honestly just never had this issue come up

**Reply by ddetton** [reply]
I thought that I saw an option to automatically create a daily note each day but now I can't find it.  

Thanks for the new youtube on outliner, dataview and templater!  Three pretty incredible additions to the Obsidian ecosystem.  

I found it.   It's in the core Daily Notes plug-in.  Are you using that or do you just do it manually?

**Reply by tallguyjenks** [reply]
i do it manually :)
